packages/astro/src/integrations/index.ts

```diff
@@ -221,17 +221,21 @@ export async function runHookConfigDone({
 								logger
 							);
 							for (const [featureName, supported] of Object.entries(validationResult)) {
-								if (!supported) {
+								// If `supported` / `validationResult[featureName]` only allows boolean,
+								// in theory 'asstes' false, doesn't mean that the feature is not supported, but rather that the chosen image service is unsupported
+								// in this case we should not show an error, that the featrue is not supported
+								// if we would refactor the validation to support more than boolean, we could still be able to differentiate between the two cases
+								if (!supported && featureName !== 'assets') {
 									logger.error(
 										'astro',
 										`The adapter ${adapter.name} doesn't support the feature ${featureName}. Your project won't be built. You should not use it.`
 									);
 								}
 							}
 							if (!validationResult.assets) {
-								logger.info(
+								logger.warn(
```

Comment:
the user shouldn't need to make any manual changes; therefore, an error is too severe